jf_agent/git/utils.py

```diff
@@ -0,0 +1,10 @@
+from jf_agent.git import NormalizedRepository
+
+# Return branches for which we should pull commits, specified by customer in git config.
+# The repo's default branch will always be included in the returned list.
+def get_branches_for_normalized_repo(repo: NormalizedRepository, included_branches: dict):
+    repo_branches = [repo.default_branch_name]
+    additional_branches_for_repo = included_branches.get(repo.name)
+    if additional_branches_for_repo:
+        repo_branches.extend(additional_branches_for_repo)
+    return repo_branches
```

Comment:
nit: might want to do `return set(repo_branches) here to eliminate dupes

Reply:
Good idea